content/pages/docs-as-code.html

```diff
@@ -129,129 +121,128 @@ <h1 class="ui huge header">
           icon=icon_1) %}
         {% markdown %}
-      Explore traffic analytics and understand what are the pages your readers read the most.
-      **Prioritize the work** to improve them before other pages with less traffic.
-      Find out what are the terms your users search the most and write documentation pages
-      that include those terms so they can easily find them.
+        Read the Docs can host multiple versions of your docs.
+        Keep your 1.0 and 2.0 documentation online,
+        so your yours can always find the right version.
         {% endmarkdown %}
+        <a class="ui basic primary button" href="https://docs.readthedocs.io/en/stable/versions.html">
+          <i class="fad fa-book icon" aria-hidden="true"></i>
+          Documentation
+        </a>
       {% endcall %}
 
     </div>
   </div>
 
   {{ about_divide() }}
 
-  <div class="ui very padded container" id="notification">
+  <div class="ui very padded container" id="{{ header_2 }}">
     <div class="ui vertically padded centered grid">
 
       {% call about.step(
           header=header_2,
-          image="/images/docs-as-code/notification.png",
-          image_alt="Notification on a old version",
+          image="/images/docs-as-code/gh-integration.png",
+          image_alt="Read the Docs checks on GitHub",
           icon=icon_2) %}
         {% markdown %}
-        Clearly communicate readers they are reading an outdated version of the documentation,
-        and **reduce support requests** from old version of your product.
+        Integrating directly with GitHub allows developers and documentation authors to collaborate seemlessly.
+        You can author your documentation in any text editor,
+        and using whatever format is most comfortable for you.
         {% endmarkdown %}
+
+        <a class="ui basic primary button" href="https://docs.readthedocs.io/en/stable/reference/git-integration.html">
+          <i class="fad fa-book icon" aria-hidden="true"></i>
+          Documentation
+        </a>
+
       {% endcall %}
 
     </div>
   </div>
 
   {{ about_divide() }}
 
-  <div class="ui very padded container" id="search">
+  <div class="ui very padded container" id="{{ header_3 }}">
     <div class="ui vertically padded centered grid">
 
       {% call about.step(
           header=header_3,
-          image="/images/docs-as-code/search.png",
-          image_alt="Read the Docs pull request review checks in a GitHub pull request",
+          image="/images/docs-as-code/visual-diff.png",
+          image_alt="Visual diff between two versions",
           icon=icon_3) %}
         {% markdown %}
-        Always find what you are looking for at the tip of your fingers with our powerful ElasticSearch engine.
-        Search on a **single or accross multiple projects** at once.
+        Once you have your documentation in Git, you can preview changes before merging them.
+        Every time you push a commit to a Pull Request,
```

Comment:
```suggestion
        Every time you push a commit to a pull request,
```

Not sure what is the thought behind this, but Anthony always propose this suggestion 😅

Reply:
"Pull request" is not a proper noun, it doesn't need proper capitalization.

GitHub uses the phrase without capitalization:

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests